docs/Changelog.md

```diff
@@ -9,6 +9,8 @@
     data type in Table API components
   * [#6552](https://github.com/TouK/nussknacker/pull/6552) Added aggregation functions to Batch aggregation component
     (`Average`, `Count`, `Max`, `Min`, `Population standard deviation`, `Sample standard deviation`, `Population variance`, `Sample variance`)
+  * [#6734](https://github.com/TouK/nussknacker/pull/6734) Available tables are now refreshed automatically when
```

Comment:
Do you mean that "is useful" is too soft? WDYTA:
>  Tables from external catalogs are now refreshed automatically when entering into node form. Please be aware that changes in `tableDefinition.sql` are not refreshed. To do this, use `/app/processingtype/reload` API

Reply:
Great :+1: Yes, imo "is useful" is too soft here since a non-experienced user could expect that when he changes the config file would cause a refresh

Reply:
Ok, changed